pom.xml

```diff
@@ -145,6 +145,18 @@
         <groupId>org.slf4j</groupId>
         <artifactId>log4j-over-slf4j</artifactId>
         <version>1.6.6</version>
+	<exclusions>
+          <!-- slf4j-api is provided by storm-core. Including slf4j-api
+	    here would cause its classes to be shaded preventing itself
```

Comment:
@erikdw Ok. So reverting `storm/pom.xml` cause MesosNimbus to throw exception like `java.lang.NoClassDefFoundError: org/slf4j/shaded/MarkerFactory` because of references like ` org.slf4j.MarkerFactory` in [here](http://grepcode.com/file/repo1.maven.org/maven2/org.slf4j/log4j-over-slf4j/1.6.6/org/apache/log4j/Category.java?av=f). So lets keep this change. I will add my reason for excluding slf4j-api in the description.